LICENSE

```diff
@@ -0,0 +1,7 @@
+## Copyright 2021, [Hassaanjbaig-code]
```

Comment:
- [ ] As requested by the previous reviewer, please setup linters correctly so that your linters checks can run. You can setup linters by copying the files from [this repo](https://github.com/microverseinc/linters-config/tree/master/html-css) into your repo and run `npm install --save-dev hint@7.x` to install webhint then run `npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x` to install stylelint.

- [Optional] Please update your pull request description to contain a summary of all changes you implemented on this branch. This will make your work more professional. Kindly check [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#5-basic-rules-to-follow-in-microverse-projects) for more information.

- [Optional] Please in all future projects, kindly make sure that your branch name is the name of a feature or action you are adding to the branch. This is best practice for [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md#feature-branches). A good branch name can be setup-mobile-portfolio.